nixos/tests/kubernetes/base.nix

```diff
@@ -42,34 +46,27 @@ let
                 };
               };
               programs.bash.enableCompletion = true;
-              environment.variables = {
-                ETCDCTL_CERT_FILE = "${certs.worker}/etcd-client.pem";
-                ETCDCTL_KEY_FILE = "${certs.worker}/etcd-client-key.pem";
-                ETCDCTL_CA_FILE = "${certs.worker}/ca.pem";
-                ETCDCTL_PEERS = "https://etcd.${domain}:2379";
-              };
+              environment.systemPackages = [ kubectl ];
               services.flannel.iface = "eth1";
-              services.kubernetes.apiserver.advertiseAddress = master.ip;
+              services.kubernetes = {
+                featureGates = ["AllAlpha"];
```

Comment:
Why this gate?

Reply:
Removed in https://github.com/NixOS/nixpkgs/pull/45670/commits/b6ea77f764b00cedd4e7783ccb9b0a87793d1b72 . Some remnant from the days where the test depended on alpha features, I guess. It doesn't anymore.